app/src/main/java/com/imashnake/animite/features/theme/Color.kt

```diff
@@ -2,67 +2,68 @@ package com.imashnake.animite.features.theme
-    Color(-6697985),
-    Color(-16764075),
-    Color(-16758152),
-    Color(-3152385),
-    Color(-16751970),
-    Color(-4602134),
-    Color(-14471091),
-    Color(-12958108),
-    Color(-2497793),
-    Color(-4013319),
-    Color(-13948071),
-    Color(-12500367),
-    Color(-1974273),
-    Color(-15393756),
-    Color(-2300946),
-    Color(-15393756),
-    Color(-2300946),
-    Color(-12629934),
-    Color(-4274220),
-    Color(-6697985),
-    Color(-2300946),
-    Color(-14012103),
-    Color(-870219),
-    Color(-10480624),
-    Color(-7594728),
-    Color(-870219),
-    Color(-7827042),
-    Color(-11975345),
-    Color(-16777216),
+    primary = Color(-6697985),
+    onPrimary = Color(-16764075),
+    primaryContainer = Color(-16758152),
+    onPrimaryContainer = Color(-3152385),
+    inversePrimary = Color(-16751970),
+    secondary = Color(-4602134),
+    onSecondary = Color(-14471091),
+    secondaryContainer = Color(-12958108),
+    onSecondaryContainer = Color(-2497793),
+    tertiary = Color(-4013319),
+    onTertiary = Color(-13948071),
+    tertiaryContainer = Color(-12500367),
+    onTertiaryContainer = Color(-1974273),
+    background = Color(-15393756),
+    onBackground = Color(-2300946),
+    surface = Color(-15393756),
+    onSurface = Color(-2300946),
+    surfaceVariant = Color(-12629934),
+    onSurfaceVariant = Color(-4274220),
+    surfaceTint = Color(-6697985),
+    inverseSurface = Color(-2300946),
+    inverseOnSurface = Color(-14012103),
+    error = Color(-870219),
+    onError = Color(-10480624),
+    errorContainer = Color(-7594728),
+    onErrorContainer = Color(-870219),
+    outline = Color(-7827042),
+    outlineVariant = Color(-11975345),
+    scrim = Color(-16777216),
 )
 
 val KimiNoLightColorScheme = ColorScheme(
-    Color(-16751970),
-    Color(-1),
-    Color(-3152385),
-    Color(-16769740),
-    Color(-6697985),
-    Color(-11379075),
-    Color(-1),
-    Color(-2497793),
-    Color(-15918281),
-    Color(-10921333),
-    Color(-1),
-    Color(-1974273),
-    Color(-15395261),
-    Color(-197377),
-    Color(-15393756),
-    Color(-197377),
-    Color(-15393756),
-    Color(-2432015),
-    Color(-12629934),
-    Color(-16751970),
-    Color(-14012103),
-    Color(-1314308),
-    Color(-5036514),
-    Color(-1),
-    Color(-401700),
-    Color(-12513781),
-    Color(-9537661),
-    Color(-3488560),
-    Color(-16777216),
+    primary = Color(-16751970),
+    onPrimary = Color(-1),
+    primaryContainer = Color(-3152385),
+    onPrimaryContainer = Color(-16769740),
+    inversePrimary = Color(-6697985),
+    secondary = Color(-11379075),
+    onSecondary = Color(-1),
+    secondaryContainer = Color(-2497793),
+    onSecondaryContainer = Color(-15918281),
+    tertiary = Color(-10921333),
+    onTertiary = Color(-1),
+    tertiaryContainer = Color(-1974273),
+    onTertiaryContainer = Color(-15395261),
+    background = Color(-197377).blendWith(Color(-16751970), 0.1f),
```

Comment:
What are you blending with
FYI you can do `0xFFAABBCC` for hexadecimal format colors

Reply:
`primary` from on top. I essentially generated the color scheme so I won't bother converting to hexadecimal.

Reply:
Wouldn't we want white (or dark grey for dark themes) to be blended at 10%?